go/pserver/client_test.go

```diff
@@ -124,3 +154,21 @@ func TestClientFull(t *testing.T) {
 		}
 	}
 }
+
+func TestNativeClient(t *testing.T) {
+	initNativeClient()
+	servers := make([]pserver.Server, numPserver)
+	for i := 0; i < numPserver; i++ {
+		servers[i] = pserver.Server{Index: i, Addr: ":" + strconv.Itoa(pserverClientPorts[i])}
+	}
+	c1 := pserver.NewClient(lister(servers), len(servers), selector(true))
+	ClientTest(t, c1)
+}
+
+//TODO(Qiao: tmperary disable etcdClient test for dependency of etcd)
+func EtcdClient(t *testing.T) {
```

Comment:
Please make sure to disable it before merging into develop branch. We need to make sure test always runs correctly, since other developers need to run test as well.